utils/unittest/fixtures.go

```diff
@@ -564,24 +565,25 @@ func IdentityListFixture(n int, opts ...func(*flow.Identity)) flow.IdentityList
 	return identities
 }
 
-func ChunkFixture() *flow.Chunk {
+func ChunkFixture(blockID flow.Identifier) *flow.Chunk {
 	return &flow.Chunk{
 		ChunkBody: flow.ChunkBody{
 			CollectionIndex:      42,
 			StartState:           StateCommitmentFixture(),
 			EventCollection:      IdentifierFixture(),
 			TotalComputationUsed: 4200,
 			NumberOfTransactions: 42,
+			BlockID:              blockID,
 		},
 		Index:    0,
 		EndState: StateCommitmentFixture(),
 	}
 }
 
-func ChunksFixture(n uint) []*flow.Chunk {
+func ChunksFixture(n uint, blockID flow.Identifier) []*flow.Chunk {
```

Comment:
Does it make sense to have multiple chunks fixtures sharing the same block ID?

Reply:
All generated chunks in `ChunksFixture` are supposed to be associated with the same block ID. This function is utilized when for example we want to create an execution result fixture with more than a chunk:
```
	return &flow.ExecutionResult{
		ExecutionResultBody: flow.ExecutionResultBody{
			PreviousResultID: IdentifierFixture(),
			BlockID:          block.Header.ID(),
			FinalStateCommit: StateCommitmentFixture(),
			Chunks:           ChunksFixture(uint(chunks), block.ID()),
		},
		Signatures: SignaturesFixture(6),
	}
```